db/seeds.rb

```diff
@@ -20,4 +20,77 @@
                email: email,
                password:              password,
                password_confirmation: password)
-end
\ No newline at end of file
+end
+
+
+# Make a rhizome
+Rhizome.create!(
+  name: "Test rhizome"
+  )
+
+ParticleDevice.create!(
+  device_id: Faker::Lorem.characters(24),
+  encrypted_access_token: "S3jF/ramSAL8dZ4zzASXeuBunWv6tLn9jzcU4ABEflfGQkrwbM44CpWMtyRhkED/",
+  rhizome_id: 1
+  )
+
+
+# Make an equipment profile
+EquipmentProfile.create!(
```

Comment:
Rather than continuously calling EquipmentProfile.find(1), maybe it would be worthwhile to capture that ID in a variable and just use the variable?